### ISSUE TITLE
how to ignore the non-css file

### ISSUE THREAD

**Issue by LiaoSept** [benbria/aliasify#34]
error info: 
```
events.js:141
      throw er; // Unhandled 'error' event
      ^

TypeError: Cannot read property './main/style/basic.css' of undefined (while aliasify was processing /home/develop/Projects/project/js/main.js) while parsing file: /home/develop/Projects/project/js/main.js (while aliasify was processing /home/develop/Projects/project/js/main.js) while parsing file: /home/develop/Projects/project/js/main.js
    at getReplacement (/home/develop/Projects/project/node_modules/aliasify/lib/aliasify.js:21:16)
    at /home/develop/Projects/project/node_modules/aliasify/lib/aliasify.js:48:21
    at /home/develop/Projects/project/node_modules/browserify-transform-tools/lib/transformTools.js:235:16
    at /home/develop/Projects/project/node_modules/browserify-transform-tools/lib/transformTools.js:176:18
    at walk (/home/develop/Projects/project/node_modules/browserify-transform-tools/node_modules/falafel/index.js:58:9)
    at /home/develop/Projects/project/node_modules/browserify-transform-tools/node_modules/falafel/index.js:55:17
    at Array.forEach (native)
    at forEach (/home/develop/Projects/project/node_modules/browserify-transform-tools/node_modules/falafel/index.js:9:31)
    at walk (/home/develop/Projects/project/node_modules/browserify-transform-tools/node_modules/falafel/index.js:43:9)
    at /home/develop/Projects/project/node_modules/browserify-transform-tools/node_modules/falafel/index.js:50:25
```
aliasify:
```javascript
// see https://github.com/benbria/aliasify#configuration
mudule.exports = {
    replacements: {
    },
    appliesTo: {
        includeExtensions: ['.js'],
        excludeExtensions: ['.css']
    },
    verbose: true
};
```

**Comment by jwalton** [comment]
Sorry, not sure I understand what you're asking.  How are you getting this error?  Do you have a sample project that recreates the problem?

**Comment by LiaoSept** [comment]
in our project there using browserify-css process including css

**Comment by LiaoSept** [comment]
@jwalton https://gist.github.com/LiaoSept/3d6f2fe2a32e1998e2c9#file-readme-md

**Comment by LiaoSept** [comment]
@jwalton

**Comment by jwalton** [comment]
There is a bug in aliasify here.  You can fix your example gist by adding an empty `aliases`:

```
module.exports = {
    aliases: {},
    replacements: {},
    appliesTo: {
        includeExtensions: ['.js'],
        excludeExtensions: ['.css']
    },
    verbose: true
};
```

I'll have a fix shortly.

**Comment by jwalton** [comment]
Fixed in 1.8.1.  If this doesn't solve your problem, please let me know. :)
